app/assets/stylesheets/shared/tooltips.scss

```diff
@@ -0,0 +1,76 @@
+.tooltip-item {  
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;  
+  $tooltip-background: white;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 16em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px silver;
+  $tooltip-distance-from-item: 5em;
+  $tooltip-arrow-distance-from-box: -1.2em;
+
+  // background: $tooltip-background;
+  // border-radius: $base-border-radius;
+  // border: 1px solid $base-border-color;
+  // display: inline-block;
+  // padding: 0.5em 1em;
+  // position: relative;
+  // text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip {
+    @include position(absolute, null 0 $tooltip-distance-from-item 0);
+    @include transition (all 0.2s ease-in-out);
+    background-color: $tooltip-background;
```

Comment:
Properties should be ordered background, background-color, border-radius, box-shadow, color, font-size, line-height, margin, max-width, opacity, padding, text-align, visibility, width, z-index